platform/web/js/libs/library_godot_input.js

```diff
@@ -133,10 +150,11 @@ const GodotIME = {
 		},
 
 		clear: function () {
-			if (GodotIME.ime) {
-				GodotIME.ime.remove();
-				GodotIME.ime = null;
+			if (GodotIME.ime == null) {
+				return;
 			}
+			GodotIME.ime.remove();
+			GodotIME.ime = null;
```

Comment:
You could also call `clearinterval` here as well since we know ime being set to null here (just check that your id exists)